scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -53,7 +53,7 @@ patches:
       path: /spec/services/0
       value: repo-setup
     - op: replace
-      path: /spec/nodeTemplate/ansible/ansibleVars/edpm_chrony_ntp_servers
+      path: /spec/nodeTemplate/ansible/ansibleVars/edpm_ntp_servers
       value:
         - ${EDPM_CHRONY_NTP_SERVER}
```

Comment:
LGTM except one thing.

On line 58 I'd replace this:
```
  - ${EDPM_CHRONY_NTP_SERVER}
```
with this
```
  - {hostname: ${EDPM_CHRONY_NTP_SERVER}}
```
Same for line 76. 

That way you don't need unnecessary extra complexity to append the hostname before the call to `linux-systemroles.timesync`. Also, people familiar with the `linux-systemroles.timesync` (since it's becoming a standard within Red Hat) can pass what they know. Also, there's no point in maintaining a semblance of backwards compatibility with the old way as it hasn't really shipped and for today's users, install_yamls hides it. I say a semblance because it's not really backwards compatible as the variable name is changing.

WDYT @jpodivin @rebtoor ?

Reply:
If there is really no need for backwards compatibility I can do that. Although I would at least like to keep the variable name, so it's obvious what kind of time servers it's setting up.

Reply:
Why we should keep backward compatibility? I mean, the old role will be deleted, i think it's better to keep 1:1 compatibility with systemrole variables, as @fultonj said.